Extractor/Pushers/InfluxPusher.cs

```diff
@@ -189,8 +197,13 @@ public InfluxPusher(ILogger<InfluxPusher> log, InfluxPusherConfig config)
 
             if (count == 0) return null;
 
+            if (fullConfig.DryRun)
+            {
+                log.LogInformation("Dry run enabled. Would insert {Count} events to influxdb", count);
+                return null;
+            }
+
             var points = evts.Select(UAEventToInflux);
```

Comment:
Is there a reason to exclude this conversion from the dry-run?

Reply:
Doesn't look like it, I'll include it.